docs/standard/microservices-architecture/secure-net-microservices-web-applications/azure-key-vault-protects-secrets.md

```diff
@@ -56,8 +56,8 @@ In this example, the call to AddAzureKeyVault comes at the end of configuration
 
 ## Additional resources
 
--   **Using Azure Key Vault to protect application secrets***
-    <https://docs.microsoft.com/azure/guidance/guidance-multitenant-identity-keyvault>*
+-   **Using Azure Key Vault to protect application secrets**
+    [*https://docs.microsoft.com/azure/guidance/guidance-multitenant-identity-keyvault*](https://docs.microsoft.com/azure/guidance/guidance-multitenant-identity-keyvault)
```

Comment:
I've previously discussed that with @BillWagner. Normally they would follow the format you're showing @svick.  However, because this content is also part of an e-book, its formatting is a bit different. For now, we're keeping like this but we might change it in the future.